index.src.html

```diff
@@ -822,10 +830,10 @@ <h4 id="clear-dom">
      
       1.  An origin's WebSQL databases [[!WEBDATABASE]].
       2.  An origin's filesystems [[!file-system-api]]
-      3.  ChannelID
```

Comment:
Didn't we argue about this? I think we need to tie Channel ID's lifetime to cookie's lifetime.